src/puptoo/utils/puptoo_logging.py

```diff
@@ -96,4 +96,9 @@ def filter(self, log_record):
         except Exception:
             log_record.account = "000001"
 
+        try:
+            log_record.org_id = threadctx.org_id
+        except Exception:
+            log_record.account = "000001"
```

Comment:
should this add a placeholder org-id instead?

Reply:
It should be `org_id` in place of `account`. Thanks!